.github/workflows/backend-ci.yml

```
@@ -0,0 +1,76 @@
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0  # Shallow clones should be disabled for better SonarCloud analysis
+        
+    - name: Set up JDK 17
+      uses: actions/setup-java@v4
+      with:
+        java-version: '17'  # SonarCloud requires Java 17+
+        distribution: 'temurin'
+        cache: maven
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)